app/controllers/sites/pages_controller.rb

```diff
@@ -24,6 +24,13 @@ def redirect
     redirect_to @news.url.blank? ? news_path(@news) : @news.url
   end
 
+  def sitemap
+    @published_menus = {}
+    current_site.components.where({name: "menu", publish: true}).each do |component|
```

Comment:
Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.